### DISCUSSION TITLE
Add calendar entry description and time to monthly view

### DISCUSSION THREAD

**Discussion by WheresWes** [totaldebug/atomic-calendar-revive#1339] [Feature Requests]
Would be wonderful to enable a view which shows the time and description of calendar entries much like is done in the calendar dashboard in home assistant.  I can see that the dots used in the current view serve a purpose on smaller displays, but offer little benefit in known 'what' or 'when' something is occurring

**Comment by marksie1988** [comment]
Hi, thanks for this suggestion. 

It's not something I had planned to implement as the card was never designed to be a large panel however as quite a few people have asked for it I may look at how I could implement such a view.

**Comment by marksie1988** [comment]
I have now added a "inline" mode, this shows a full calendar view designed for use in a panel.
